FSHD_CLI/README.md

```diff
@@ -0,0 +1,88 @@
+
+
+
+# Haplotype Predictor
+
+This Python script allows you to predict genotypes from Short Sequence Length Polymorphisms (SSLPs) values for Facioscapulohumeral muscular dystrophy (FSHD) analysis. It requires a selection of SSLPs and the region from where the sample is derived.
+
+## Pip package
+This analayser can also be used as a standalone pip package. It can be installed by navigating to the FSHD_CLI directory and executing the following command in your enviroment of choice.
+```bash
+pip install .
+```
+
+The package can then be used as normal by substituting `python FSHD.py` with `FSHD`
+
+## Example
+```bash
+FSHD -s <SSLP1 SSLP2 SSLP3 SSLP4> -p <population>
+```
+
+## Requirements
+
+    Python 3.x
+    - sys
+    - json
+    - shutil
+    - argparse
+    - pkg_resources
+    - questionary
+    - pathlib
+    - itertools
+
+## Script Description
+
+The script uses pre-defined haplotypes for prediction. A haplotype is a group of genes within an organism that was inherited together from a single parent. This script uses haplotypes of certain populations (European, African, Asian) to predict the genotypes from the provided SSLP values.
```

Comment:
This definition of a haplotype isn't fully correct - please have a look.
Also, the script is actually trying to predict the exact haplotypes present in the patient. Genotypes isn't wrong, but I don't feel the sentence fully explains the script's function.